### PR TITLE
Improve storybook

### PR DESCRIPTION
**Pull request by ganlhi**
This PR is about adding useful code samples to Storybook so it can be considered a _developer oriented_ documentation. 
For instance, for the `Card` component: 

image

⚠️ It is a work in progress, so it shouldn't be merged yet.

### CODE REVIEW

File: src/components/Overlay/index.stories.js [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-));
+stories.add('Playground', () => {
+  const isOpen = boolean('Is Open', true);
+  const animateOnMount = boolean('Animate On Mount', false);
+  return (
+    <>
+      <Heading>Overlay</Heading>
+      <Overlay isOpen={isOpen} animateOnMount={animateOnMount} />
+
+      <Code>{`<Overlay isOpen={${isOpen}} animateOnMount={${animateOnMount}}/>`}</Code>
```
Comment (tamarasaurus) [inline src/components/Overlay/index.stories.js:19]: Thanks for documenting this extra option :pray:

File: src/components/Pagination/index.stories.js [hunk 2]
```diff
@@ -2,26 +2,70 @@
...
+      `}</Code>
+
       <Pagination
         prevDisabled={page === 0}
         nextDisabled={page === 3}
         goNext={() => setPage(p => p + 1)}
         goPrev={() => setPage(p => p - 1)}
-        position={select('Position', ['flex-end', 'flex-start', 'center'])}
+        position={position}
+        translate={translate}
```
Comment (tamarasaurus) [inline src/components/Pagination/index.stories.js:68]: I had no idea this was an option :sweat_smile:

**Review by tamarasaurus — APPROVED** [review]
Thanks for making these changes :clap: The Storybook is significantly more useful now :slightly_smiling_face:

File: src/components/Pagination/index.stories.js (hunk 2 above)
Comment (ganlhi) [reply src/components/Pagination/index.stories.js:68]: Working on Driver Documents, I said that exact same phrase to myself a bunch of times. That's why this PR exists ^^

**Review by keanan — APPROVED** [review]

**Pull request merged**